api/Element.json

```
@@ -1674,12 +1674,28 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/className",
           "support": {
-            "chrome": {
-              "version_added": "43"
-            },
-            "chrome_android": {
-              "version_added": "43"
-            },
+            "chrome": [
+              {
+                "version_added": "22"
+              },
+              {
+                "version_added": "1",
+                "version_removed": "22",
+                "partial_implementation": true,
+                "notes": "This function is implemented in the <code>HTMLElement</code> API, meaning non-HTML elements (like SVG elements) cannot use this function."
```

```suggestion
                "notes": "Only supported for <a href='https://developer.mozilla.org/docs/Web/API/HTMLElement'><code>HTMLElement</code></a>, not all <code>Element</code> objects, such as <a href='https://developer.mozilla.org/docs/Web/API/SVGElement'><code>SVGElement</code></a>."
```

Suggesting something closer to the wording used in https://github.com/mdn/browser-compat-data/pull/6865. (Also note that this is an attribute, not a function.)

Having written that out, I'd also suggest testing if this was ever added to `SVGElement` specifically. A bunch of things were at one point on `HTMLElement` + `SVGElement` and then moved up to `Element`, which would make the note wrong